baseclasses/testing/comparisons.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+
+DEFAULT_TOL = 1e-12
+
+
+def assertEqual(a, b):
+    """
+    This replaces the assertEqual functionality provided by unittest.TestCase
+    so that we can call this outside the class
+    """
```

Comment:
Sounds good but I can't picture cases where this is used. Are you thinking about using this outside testing?

Reply:
Well, I guess there are two reasons for doing this. The first is as stated---this allows people to use this function elsewhere, though I guess that' not that common. The other reason is that, otherwise, we have to subclass `unittest.testcase`, and overload this function. That seems like a lot of work so I'd rather just define this elsewhere.

Reply:
These were copied directly from pyOptSparse FYI